### DISCUSSION TITLE
How does utils.summary_data_from_transaction_data calculate monetary value?

### DISCUSSION THREAD

**Discussion by gititinyoursoul** [CamDavidsonPilon/lifelines#1243] [Q&A]
Hi there,

in the [documentation ](https://lifetimes.readthedocs.io/en/latest/Quickstart.html#the-shape-of-your-data) it explicitly says: '(monetray_value) is equal to the sum of all a customer’s purchases divided by the total number of purchases. Note that the denominator here is different than the frequency described above'. 

After some confusion, I found that to be not true and the utility function divides the sum of all a customer’s purchases by the frequency count.

Is this the documentation not correct?

**Comment by CamDavidsonPilon** [comment]
Hey, this is the lifelines repo, not the lifetimes repo

**Reply by gititinyoursoul** [reply]
haha, my bad :)
